cli/command/swarm/opts.go

```diff
@@ -3,9 +3,12 @@ package swarm
 import (
 	"encoding/csv"
 	"fmt"
+	"io/ioutil"
 	"strings"
 	"time"
 
```

Comment:
Left one blank line between the stdlib imports and the `github.com/*` imports, but removed the extraneous ones between the stdlib imports.